tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -57,7 +57,7 @@ TEST(RNTupleInspector, CompressionSettings)
       auto nFldInt = model->MakeField<std::int32_t>("int");
 
       auto writeOptions = RNTupleWriteOptions();
-      writeOptions.SetCompression(505);
+      writeOptions.SetCompression(207);
```

Comment:
Any specific reason to change the compression options here?

Reply:
The default compression setting is already 505, so I figured it's redundant to explicitly set it here. The alternative would have been to remove this line altogether, that would also be fine.

Reply:
> The default compression setting is already 505

But it might change in the future :)

Reply:
Fair enough! That's why right now I opted for the current change. I can also revert this change altogether, at the time I made the change it made sense to me but it doesn't really matter after all.